src/asim/configs/resident/vehicle_type_choice_op4.csv

```diff
@@ -1,13 +1,13 @@
 ﻿Label,Description,Expression,Coefficient
-util_ln_nmods,number of models available,"@np.log(1 + df.NumModels)",coef_ln_nmods
-util_ln_nmakes,number of makes available,"@np.log(1 + df.NumMakes)",coef_ln_nmakes
+util_ln_nmods,number of models available,"logged_models",coef_ln_nmods
+util_ln_nmakes,number of makes available,"logged_makes",coef_ln_nmakes
 util_mpg,miles per gallon (or equivalent),"@df.MPG",coef_mpg
 util_crange,Range for BEV (mi),"@df.Range",coef_crange
-util_crangeltwk,range less than average round trip distance to work,"@np.where((df.Range < df.avg_hh_dist_to_work * 2) & (df.fuel_type == 'BEV'), 1, 0)",coef_crangeltwk
-util_ln_chpc_ev,ln(1+number of chargers per capita in MSA/state),"@np.log(1+CHARGERS_PER_CAP) * ((df.fuel_type == 'PEV') | (df.fuel_type == 'BEV'))",coef_ln_chpc_ev
```

Comment:
These operations where you remove the string comparisons needs to happen for ALL off the expressions in this file. I suggest starting from the configs [here](https://github.com/ActivitySim/activitysim-prototype-mtc/pull/3/files) and then adding the additional SANDAG terms like AV and SAN-specific constants back in.

Reply:
I had done that (using those configs as a reference), but I may not have properly committed it... It should be fixed now (and I removed all of the non-SAN region-specific terms as well)